frontend/src/component/project/ProjectCard/ProjectCard.styles.ts

```diff
@@ -22,22 +18,32 @@ export const StyledProjectCard = styled(Card)(({ theme }) => ({
         transition: 'background-color 0.2s ease-in-out',
         backgroundColor: theme.palette.neutral.light,
     },
+    borderRadius: theme.shape.borderRadiusMedium,
 }));
 
-export const StyledDivHeader = styled('div')(() => ({
+export const StyledProjectCardBody = styled(Box)(({ theme }) => ({
+    padding: theme.spacing(1, 2, 2, 2),
+}));
+
+export const StyledDivHeader = styled('div')(({ theme }) => ({
     ...flexRow,
     width: '100%',
+    marginBottom: theme.spacing(2),
 }));
 
 export const StyledH2Title = styled('h2')(({ theme }) => ({
     fontWeight: 'normal',
     fontSize: theme.fontSizes.bodySize,
-    lineClamp: 2,
+    lineClamp: '2',
+    WebkitLineClamp: 2,
+    lineHeight: '1.2',
     display: '-webkit-box',
     boxOrient: 'vertical',
     textOverflow: 'ellipsis',
     overflow: 'hidden',
     alignItems: 'flex-start',
+    WebkitBoxOrient: 'vertical',
```

Comment:
This is deprecated ([according to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient)). Can we use a different property?